src/ro/catalog/servicii/Audit.java

```diff
@@ -0,0 +1,39 @@
+package ro.catalog.servicii;
+
+import java.io.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Audit {
+    public static Audit auditService;
+
+    private Audit() {}
+
+    public static Audit getAudit() {
+        if (auditService == null)
+            auditService = new Audit();
+        return auditService;
+    }
+
+    public void writeAction(String action) throws IOException {
```

Comment:
redundant declaration of throws IOException, since you treat it (declare it in catch)

Reply:
Fixed.